CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Changelog
+## [0.4.0] - June 3, 2024
+- Added `lce-flow` module for Coroutine Flow support.
```

Comment:
Nit: `lce-coroutines`

Reply:
Good catch, I was back and forth on the name of this. Fixed.